packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -67,7 +67,7 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
   )
 
   const approvalFeeText = useMemo(() => {
-    const eth = formatAmount(estimatedGasFees, { symbol: 'ETH' })
+    const eth = formatAmount(estimatedGasFees, { symbol: ether.symbol })
```

Comment:
This will only needed to be changed after we have custom fee token networks that are parent chains